server/src/session.rs

```diff
@@ -34,12 +34,20 @@ impl Session {
         Self {
             path: OwnedObjectPath::try_from(format!("/org/freedesktop/secrets/session/s{index}"))
                 .unwrap(),
-            _aes_key: aes_key,
+            aes_key,
             manager,
         }
     }
 
     pub fn path(&self) -> &OwnedObjectPath {
         &self.path
     }
+
+    pub fn aes_key(&self) -> Option<&Key> {
+        if let Some(key) = &self.aes_key {
+            Some(key.as_ref())
+        } else {
+            None
+        }
+    }
```

Comment:
Return a Option<Arc<Key>> like you did for session.

Reply:
Done, thanks!